diff --git a/tsrc/id/windump/StackWalker.cpp b/tsrc/id/windump/StackWalker.cpp
index 3333333..4444444 100644
--- a/tsrc/id/windump/StackWalker.cpp
+++ b/tsrc/id/windump/StackWalker.cpp
@@ -650,6 +650,17 @@ BOOL SomeClass::LoadModules(HANDLE hProcess)
+      if (vData != NULL)
+      {
+        free(vData);
+      }
+      switch(Module.SymType)
+      {
+        case SymNone:
+          szSymType = "-nosymbols-";
+          break;
+        case SymPdb:
+          szSymType = "PDB";
+          break;
+      }
