diff --git a/tsrc/id/iduLimitTest/iduLimitTest.cpp b/tsrc/id/iduLimitTest/iduLimitTest.cpp
index 7777777..8888888 100644
--- a/tsrc/id/iduLimitTest/iduLimitTest.cpp
+++ b/tsrc/id/iduLimitTest/iduLimitTest.cpp
@@ -120,5 +120,14 @@ IDE_RC StartThread()
+    for(i = 0; i < IDU_TEST_THREAD_COUNT; i ++)
+    {
+        if(gThreadExist[i] == ID_FALSE)
+        {
+            continue;
+        }
+        IDE_TEST(gThread[i].start() != IDE_SUCCESS);
+        idlOS::printf("Thread [" ID_INT32_FMT "] Created Successfully!\n", i);
+    }
